sass/layout/_markdown.scss

```diff
@@ -57,12 +57,18 @@ h6 {
         margin: 5px 0;
         max-height: 500px;
     }
-    div.markdown-inline-img {
+    div .markdown-inline-img {
```

Comment:
The style was inactive, not sure if that was intentional. I've fixed/activated it